### PR TITLE
Bottlenecking/onedrive article

### PR DESCRIPTION
**Pull request by Poodila**
new bottlenecking article

### CODE REVIEW

File: docs/learning/Bottlenecking.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+layout: default
+title: Bottle Necking
+nav_exclude: false
+has_children: false
+parent: Learning
+search_exclude: false
+last_modified_date: 2023-02-27
+---
+
+# BottleNecking
```
Comment (sealsrock12) [inline docs/learning/Bottlenecking.md:11]: Is it Bottle necking, BottleNecking, or Bottlenecking? You should use one for everything.

File: docs/guides/OneDrive.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+# Unsyncing/Uninstalling One Drive from your System
+
+## Unsyncing for a short period of time
+If you want to stop syncing OneDrive on your computer because you are on a metered or slow internet connection, you can Pause and Resume your sync. Pausing and Resuming can also help if OneDrive status displays "Processing changes" for a long time.
+
+### To pause syncing
+1. Select the white or blue OneDrive cloud icon in the notification/menu area. This icon may be under the "show hidden icons"tab on your taskbar. (See pictures for refrencing below)
+![Show Hidden Icons](https://imgur.com/9Eibny8.png) 
+![One Drive Icon](https://imgur.com/XPr89uv.png)
+2. Right-click (or CTRL-click in Mac) the OneDrive icon on the taskbar, Click the        ![settings](https://support.content.office.net/en-us/media/9c35458d-dc74-4d37-9cc7-89aff307eff9.png) icon and select Pause syncing > 2 / 8 / 24hours.
```
Comment (K97i) [inline docs/guides/OneDrive.md:20]: You could upload the images under the assets folder in the main folder. Just remember to make your own folder for the specific article. If you want, you can add sub folders for each OS for your convenience.

File: docs/learning/Bottlenecking.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Bottle Necking
```
Comment (K97i) [inline docs/learning/Bottlenecking.md:3]: Like @sealsrock12 said, make sure your titles are **_actually consistent._**

Comment (K97i) [reply docs/learning/Bottlenecking.md:3]: (use **Bottlenecking**)

**Review by K97i — CHANGES_REQUESTED** [review]

**Pull request merged**